VirtoCommerce.CustomerModule.Data/Search/Indexing/MemberDocumentBuilder.cs

```diff
@@ -162,5 +168,30 @@ protected virtual void AddParentOrganizations(IndexDocument document, ICollectio
             document.AddFilterableValues("ParentOrganizations", nonEmptyValues);
             document.AddFilterableValue("HasParentOrganizations", nonEmptyValues?.Any() ?? false);
         }
+
+        protected virtual void IndexDynamicProperty(IndexDocument document, DynamicObjectProperty property)
+        {
+            var propertyName = property.Name?.ToLowerInvariant();
+            if (!string.IsNullOrEmpty(propertyName))
+            {
+                var isCollection = property.IsDictionary || property.IsArray;
+                var values = new List<object>();
+
+                if (property.IsDictionary)
+                {
+                    //add all locales in dictionary to searchIndex
+                    values.AddRange(property.Values.Select(x => x.Value)
```

Comment:
replace with `values = property.Values.Select(...).ToList();`

Reply:
Anyway it will call AddRange() inside .ToList(), but if you think it's good for code readability.